Fixed odd hunger bar glitch

MystWraith opened the pull request:
Also made the code a lot simpler and compact.

This was to address the glitch in #15.

If more or less hunger bars are desired, it is possible. Simply change the `return 9` to any other number between `0` and `10`. The hunger bars fill from right to left, so higher numbers mean filling from left to right.

MystWraith committed:
Fixed odd hunger bar glitch

Also made the code a lot simpler and compact.

MystWraith committed:
Removed useless comment

MystWraith committed:
Removed useless log param

showierdata9978 commented:
![2023-08-22_18 44 17](https://github.com/showierdata9978/Pickaxe-mod/assets/68120127/5da6cd90-4b0c-4691-8028-f0c8867e2757)

MystWraith commented:
Do you not want it to show at all? I can do that too. It's a simple change

showierdata9978 commented:
o wait, i see you set it to 9 by default xd, but yeah it should be gone

showierdata9978 committed:
Change to 10

MystWraith commented:
<img width="252" alt="image" src="https://github.com/showierdata9978/Pickaxe-mod/assets/77246251/dbbe2276-e94e-4445-a556-4eaf80bbbfd5">
Forgot about this

MystWraith committed:
Variety of updates

- Added a removal of the mount health bar.
- Added a removal of the breath bar

MystWraith commented:
This should now remove the hunger bar, the mount bar, and the breath bar.

MystWraith committed:
No prints, sorry

MystWraith committed:
Alt method comment

The comment that was added was an alternate way to do roughly the same thing, but there are interesting consequences.

MystWraith committed:
Wrong one commented, mb

MystWraith commented:
The alt method looks like this:
<img width="203" alt="image" src="https://github.com/showierdata9978/Pickaxe-mod/assets/77246251/8809ba6c-89b8-4910-98dc-67bf2a908101">
Which is a bit overkill.
Plus I think it kills the XP bar effects.
So if I made it a tiny bit more stable it's possible, but otherwise, it is not viable (yet).

MystWraith commented:
Possibly in the future I will change the `PlayerEntity` variable to have it's hunger and air values set to 0 at all times in pickaxe, but for now I'll stick with the other method.

showierdata9978 commented:
Health is important

showierdata9978 commented:
Hunger value must remain more then 3 (in the sat/huger variable) for sprinting

MystWraith commented:
Gotcha, so no editing the PlayerEntity variable. Understood. I was just looking for a way to avoid the `14` ordinal

MystWraith commented:
I think this is as good as I am going to get. My worry about the `ordinal = 14` is unfixable with the current abilities of mixins. So it is complete.
